priv/templates/nimble_template/.github/workflows/test.yml.eex

```diff
@@ -87,22 +144,254 @@ jobs:
 
       - name: Run codebase check
         run: mix codebase
+    
+  seed_data:
+    name: Test seed data
```

Comment:
To make it clearer, I recommend naming this:

```suggestion
    name: Test database seeds
```

It is a small change that makes it clear IMO

Reply:
Updated in https://github.com/nimblehq/elixir-templates/pull/163/commits/8c6197c1f715061cc3261c1ac2c284a90c71e079